### DISCUSSION TITLE
Solução do exercício 3 do capítulo 3 rodando no Blender 2.93.1

### DISCUSSION THREAD

**Discussion by chicortiz** [clube-de-leitura-python/pense-em-python-2021#21] [Mostre alguma coisa]
Oi pessoal!

Uma curiosidade: consegui usar o Blender 3D versão 2.93.1 para rodar o exercício 3 do capítulo 3. É necessário no entanto iniciar no terminal o aplicativo para usar o mesmo como "Shell". No Windows deve aparecer no prompt de comando o desenho.

[Imagem](https://1.bp.blogspot.com/-dNnPDZaj7CE/YSzvklZu3fI/AAAAAAAAFxM/lp_zFd3d7IcqAcZzaLaNi__qhNkpVykGgCLcBGAsYHQ/s1246/ScriptingBlender2931.png)

Link para o arquivo com a interface do Blender bem limpa:

[Link para download do arquivo .blend](https://drive.google.com/file/d/1BoEUUBBe3Y8I636OQEWi-HE9zoXBfGOO/view?usp=sharing)

Você pode fazer o download do Blender aqui:

[Blender.org](https://www.blender.org/)

Saudações,

Ortiz

**Comment by villares** [comment]
Dá pra postar imagens aqui no fórum, só dar "colar/paste":

![image](https://user-images.githubusercontent.com/3694604/131360652-2cd951c8-8f7d-457d-b0d0-5c837725900f.png)

Próximo passo é uma janela em 3D mesmo!
